### DISCUSSION TITLE
Battery Level Warning - Where to adjust?

### DISCUSSION THREAD

**Discussion by meggenberger** [teslamate-org/teslamate#1553] [Q&A]
I have a 2021 Tesla 3 and Tesla recommends charging up to 90%. Now with the current settings anything above 81% charging is displayed yellow in the Battery Level Panel and >91% is red.
Can this be changed somehow? I see the values when I view the panel JSON but I do not see an option to edit it directly.

**Comment by adriankumpf** [comment]
Yes, you can customize all dashboards to your liking. To do this, first [log in with the default user](https://grafana.com/docs/grafana/latest/getting-started/getting-started/#step-2-log-in). The buttons for editing will then become visible.

**Reply by bahree** [reply]
So one question, which I had edited in the past, but I can't remember now. When I edit this, how do I save it so it persists:

![image](https://user-images.githubusercontent.com/3529468/115445914-53b90700-a1cb-11eb-83c5-d0d817c21c52.png)

**Reply by adriankumpf** [reply]
This requires creating a new dashboard with the customized JSON.
